politeiawww/ticketvote.go

```diff
@@ -73,7 +93,7 @@ func (p *politeiawww) voteAuthorize(a ticketvote.Authorize) (*ticketvote.Authori
 func (p *politeiawww) voteDetails(token string) (*ticketvote.DetailsReply, error) {
```

Comment:
This function should be updated to be a batched call since the plugin command is now a batched call.